draft-ietf-masque-h3-datagram.md

```diff
@@ -180,56 +161,58 @@ of {{QUIC}}):
 
 ~~~
 HTTP/3 Datagram {
-  Quarter Stream ID (i),
+  Encoded Stream ID (i),
   [Context ID (i)],
   HTTP Datagram Payload (..),
 }
 ~~~
 {: #h3-datagram-format title="HTTP/3 DATAGRAM Format"}
 
-Quarter Stream ID:
-
-: A variable-length integer that contains the value of the client-initiated
-bidirectional stream that this datagram is associated with, divided by four (the
-division by four stems from the fact that HTTP requests are sent on
-client-initiated bidirectional streams, and those have stream IDs that are
-divisible by four). The largest legal QUIC stream ID value is 2<sup>62</sup>-1,
-so the largest legal value of Quarter Stream ID is 2<sup>62</sup>-1 / 4.
-Receipt of a frame that includes a larger value MUST be treated as a connection
-error of type FRAME_ENCODING_ERROR.
+Encoded Stream ID:
+
+: A variable-length integer that contains an encoded value indicating the
+client-initiated bidirectional stream that this datagram is associated with.
+The Encoded Stream ID for a datagram without a context ID is the associated
+stream ID divided by two. The Encoded Stream ID for a datagram with a context
+ID is the associated stream ID divided by two, plus one. (Client-initiated
+bidirectional streams have stream IDs that are divisible by four, so the
```

Comment:
so what does a value of 0 mean?  why divisible by four, but only divide by two?  how is this related to the other simplifications?